src/software/ai/motion_constraint/motion_constraint_set_builder.cpp

```diff
@@ -56,9 +56,12 @@ std::set<MotionConstraint> buildMotionConstraintSetFromGameState(
     }
     else if (game_state.isKickoff())
     {
-        motion_constraints.insert({MotionConstraint::HALF_METER_AROUND_BALL,
-                                   MotionConstraint::CENTER_CIRCLE,
-                                   MotionConstraint::ENEMY_HALF});
+        motion_constraints.insert(
```

Comment:
I'm not sure why these changes are needed

Reply:
This was in the TODO in the ticket, we should ask Arun.

Reply:
I see, ignore my comment then